src/plugin/elan.js

```diff
@@ -237,7 +237,6 @@ export default class ElanPlugin {
             const th = document.createElement('th');
             th.className = 'wavesurfer-tier-' + tier.id;
             th.textContent = tier.id;
-            th.style.width = this.params.tiers[tier.id];
```

Comment:
How about
```
if (this.params.tiers) { th.style.width = this.params.tiers[tier.id]; }
```
?

Reply:
Sure.  Again, the best I can tell `th.style.width` is not used and it doesn't matter what we set it to as long as it doesn't throw an error.

I updated the PR with this suggested edit.

Reply:
Thanks! I don't remember now why it was there. I assume it sets the width of the columns that way. Otherwise they would depend on the contents of the cells.

Reply:
Right, but the value it gets set to is `true` and I'm not sure what that means in the CSS context of `th.style.width`.  It doesn't appear to harm anything, but wanted to mention it.  This is fine with me as long as it is not throwing errors.

Reply:
True, in example/elan it's passed as true. Maybe it was meant to accept a CSS value and is wrong in the example? I don't remember. Either way, both deleting it entirely and making it optional work. Let's just keep it optional. 👍